### PR TITLE
`saleor login` should give additional info 

### PR DESCRIPTION
**Pull request by Cloud11PL**
## Related issues

- https://github.com/saleor/saleor-cli/issues/348

## Steps to test feature

- 

## I have:

- [x] Tested it locally and it doesn't break existing features
- [ ] Added documentation if public changes are introduced
- [ ] Added tests for my code

### CODE REVIEW

File: src/cli/info.ts [hunk 1]
```diff
@@ -52,4 +54,19 @@ export const handler = async (): Promise<void> => {
     chalk.blue('Github  - https://github.com/saleor/'),
     'https://github.com/saleor/'
   );
+
+  console.log('');
+
+  try {
+    const { token } = await Config.get();
+    const user = (await GET(API.User, { token })) as any;
```
Comment (lkostrowski) [inline src/cli/info.ts:62]: Please don't use `any`. You can do `as unknown`, then check if `user.email` exist. If API for some reason changes, `user.email` will crash application

**Review by lkostrowski — CHANGES_REQUESTED** [review]

File: src/cli/info.ts (hunk 1 above)
Comment (Cloud11PL) [reply src/cli/info.ts]: I based it on other usages of `GET(API...)` in cli, all of them cast it to `any`. I added a check func here

**Review by lkostrowski — APPROVED** [review]

File: src/cli/info.ts [hunk 2]
```diff
@@ -52,4 +58,21 @@ export const handler = async (): Promise<void> => {
...
+
+  console.log('');
+
+  try {
+    const { token } = await Config.get();
+    const user = (await GET(API.User, { token })) as unknown;
+
+    if (hasEmail(user)) {
+      console.log(chalk.green(`Hello ${user.email}, you're logged in`));
+    }
```
Comment (lkostrowski) [inline src/cli/info.ts:70]: @zaiste here in else we can report error to sentry, so we immediately know something broke

**Comment by zaiste** [comment]
@Cloud11PL can you rebase?

**Review by zaiste — APPROVED** [review]

**Pull request merged**